README.md

```diff
@@ -1,7 +1,33 @@
+# OpenShift Serverless Previews Documentation
 
-# Knative Eventing on OpenShift Container Platform <img src="https://github.com/openshift-knative/docs/blob/master/images/knative-openshift-logo.png" width="100" height="50" />
+OpenShift Serverless Previews provides tutorials and information about
+additional, advanced use cases and integrations that are possible using
+OpenShift Container Platform and OpenShift Serverless, but are for development
+purposes only and are not officially supported by Red Hat.  This repository
```

Comment:
I don't think this is really accurate, it's not advanced use cases or integrations, just developer preview features for OpenShift Serverless

Reply:
Fair enough. I was just copying from https://openshift-knative.github.io/docs/docs/index.html. I'll change that page as well.